logs_add.php

```diff
@@ -15,7 +15,12 @@
         
         $q = "INSERT INTO logs(area,activity,people,doe) VALUES('$area','$activity','$people','$sd')";
         $r = mysqli_query($db,$q);
-        
+        $c="SELECT * from logs where activity='$activity' and cast(doe as date)='$sd'";
```

Comment:
performing a check after the insert operation is useless

Reply:
ok noted